app/src/main/java/com/springcamp/rostykboiko/rada3/EditorContract.java

```diff
@@ -9,12 +16,18 @@
 
         String getSurveyTitle();
 
+        android.view.View findViewById(@IdRes int id);
```

Comment:
You could use `ButteKnife lib` insread of android.view.View findViewById(@IdRes int id);